docs/Changelog.md

```diff
@@ -52,12 +53,11 @@ and `GenericNodeTransformation.implementation` is not invoked in this case
 * [#2349](https://github.com/TouK/nussknacker/pull/2349) Removed module `queryable-state`, `FlinkQueryableClient` was moved to `nussknacker-flink-manager`.
 * `PrettyValidationErrors`, `CustomActionRequest` and `CustomActionResponse` moved from `nussknacker-ui` to `nussknacker-restmodel`.
 * [#2361](https://github.com/TouK/nussknacker/pull/2361) Removed `security` dependency from `listener-api`. `LoggedUser` replaced with dedicated class in `listener-api`.
-* [#2367](https://github.com/TouK/nussknacker/pull/2367), [#2406](https://github.com/TouK/nussknacker/pull/2406) Simple kafka-based streaming scenario interpreter. 
+* [#2367](https://github.com/TouK/nussknacker/pull/2367) Simple kafka-based streaming scenario interpreter. 
```

Comment:
why u remove it and below?

Reply:
Thank's for catching this one.
It wasn't intended, just a mistake on rebasing.

Reply:
fixed